### DISCUSSION TITLE
pom.xml error

### DISCUSSION THREAD

**Discussion by Manvendra1097-eng** [akash-coded/spring-framework#78] [Q&A]
![errorPom xml](https://user-images.githubusercontent.com/69745291/168483426-6e17270d-7c37-4868-884c-4d16e2e9943b.png)

please help with this error @akash-coded

**Comment by akash-coded** [comment]
@Manvendra1097-eng Did it get resolved?

**Reply by Manvendra1097-eng** [reply]
yes, I have reinstall Eclipse it's resolved

**Comment by akshaychromule61** [comment]
@akash-coded I unable open pom.xml. Getting 'Failed to create the part's controls'. So I deleted it and created once again.
Also getting the error Failed to create the part's controls in driver class. I forcefully imported the import org.springframework.context.support.AbstractApplicationContext;
 packages

**Reply by raisyraphi** [reply]
I am also unable to open pom.xml

**Reply by akash-coded** [reply]
@akshaychromule61 @raisyraphi Please refer to the [answer](https://github.com/akash-coded/spring-framework/discussions/82#discussioncomment-2794608) in discussion #82 for the resolution. The error is because of missing tm4e, which you have to install in Eclipse.
